### ISSUE TITLE
Keyboard navigation does not work in ListBox

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#7837]
**Case 1** 
- expand the dropdown (click on the dropdown handle) or insert a white space
 Keyboard navigation does not work in the expanded ListBox

**Case 2** 
- expand the dropdown (click on the dropdown handle) or insert a white space
- insert `default` text inthe filter input
 **BUG**  - `No matching items` message appears 
![image](https://github.com/user-attachments/assets/2b54f655-a671-4442-b055-744ea41b8852)
